### PR TITLE
initialized Java project

### LINKED ISSUE #9: [Dev] Java Project Initialization

### PR DESCRIPTION
**Pull request by FullSol**
closes #9

### CODE REVIEW

**Comment by FullSol** [comment]
We will discuss reviewing later today. **DO NOT REVIEW AT THIS TIME** (As in don't take any actions, feel free to look at the files).

File: HELP.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Read Me First
```
Comment (L3viathon) [inline HELP.md:1]: Was thinking about creating a resources file. You beat me to it @FullSol !

**Review by L3viathon — DISMISSED** [review]
Amazing! Thank you for doing this.

**Review by briiitelord — DISMISSED** [review]

**Review by L3viathon — APPROVED** [review]
Great!

File: target/classes/application.properties [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.username=dev
+spring.datasource.password=postgres
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.h2.console.enabled=true
+
+spring.datasource.hikari.connectionTimeout=20000
+spring.datasource.hikari.maximumPoolSize=5
+spring.jpa.hibernate.ddl-auto=create
```
Comment (pyaeger) [inline target/classes/application.properties:23]: Looks good. Wizard stuff n' all.

**Review by pyaeger — APPROVED** [review]
Review finished. Approved.

**Pull request merged**